### ISSUE TITLE
NTR:  model system study evidence

### ISSUE THREAD

**Issue by mgiglio99** [evidenceontology/evidenceontology#254]
We have 'animal model system study evidence' ECO:0000179
Seems we should have the analogous plant term or a more generic model system term to encompass all types of species. (Since making specific terms for all types of organisms will be tricky.) We could then merge the animal term into the generic term.

**Comment by ivanerill** [comment]
Makes sense to me. Maybe one for each major branch of life (e.g. plant, microbial, fungi)

**Comment by mgiglio99** [comment]
Hi, Suvvi and I were looking at this one again. We haven't found any gene annotations to the "animal model system study evidence" term. I think we did use it in the CollecTF annotations.
We are having hard time thinking of when this term would be used as opposed to a more specific evidence term.
We are thinking of perhaps changing this to be a more generic term such as "model organism system study evidence" - that way it's still around as a concept but we avoid making additional organism-type terms.
@ivanerill what do you think?

**Comment by ivanerill** [comment]
I can't remember having used it, and I really can't think of a clear-cut application for it. I believe that right now replacing it for the more generic instance makes sense

**Comment by pgaudet** [comment]
Hi @mgiglio99 
I think we had requested that. The use case was test in animals that simulated clinical studies in human. How would you annotate this ?

**Comment by mgiglio99** [comment]
Hi @pgaudet ,
I've been trying to think through these and I'm not sure.
Let's imagine a mouse model for Alzheimer's and in the experiment brain tissue is taken from a mouse and it's found that a pathway is functioning differently in mice with Alzheimer's and mice without. And there are orthologs in human for the genes involved. Would the evidence be ISO or would it be better to use the animal model term above? Both are useful I think. 
I'm not sure this is a good example.

**Comment by pgaudet** [comment]
Hi Michelle, 

This is the type of paper we annotated with this evidence: PMID:22302819

The statement we made was: Gene G is 'relevant to a disease model' for Disease D. 
Probably ISO would be OK (although some animal models may not always have mutations in the same genes as the corresponding human disease). 

Maybe @nnouspike has comments/suggestions for this. 

Thanks, Pascale

**Comment by ivanerill** [comment]
So the goal here is to use an animal model as surrogate for (typically) a human, as in inferring that a gene has a role in disease development in the rat and, therefore, it likely does so in humans. Formally this is equivalent to detecting (using some experiment) that a gene is relevant for something (say disease) in an animal model and then establishing a 1:1 orthology relationship between the animal model and human genes. 

If the intention of ECO:0000179 is to indicate a standardized process for the second part of this setup (i.e. a specific pipline for establishing homology between human and rat genes), then it seems to me that it should be a child of ECO:0000044.

If the aim of ECO:0000179 is to bundle up the entire experiment+orthology setup, then it should be a child of ECO:0000006, as it is now. I wonder, though, whether it makes sense to have such a term, instead of the double annotation for experiment + orthology (or simply annotating using a cross-product of the two evidence terms). Using only ECO:0000179 as evidence in this context may make life easier for the curator, but in a way it _hides_ the type of experimental evidence used (e.g. IDA, IMP...) under the shroud of "animal model".

**Comment by mgiglio99** [comment]
Hi @pgaudet 

I'm revisiting old issues and wanted to follow up on this one. Sorry for letting it sit so long. 

I looked at the paper you mentioned above and I just want to make sure I understand what Gene G is in your comment.
Disease D is clearly Rett syndrome. 
I feel the 'is relevant to a disease model' is a bit of an ambiguous relationship. Is it designed to be that way so it could be used for either the genes in the model or the genes in the organism with the actual disease?
 
If Gene G is the mouse gene - either MeCP2 or TrkB from mouse - then I'd think the evidence for the above statement, e.g. TrkB 'is relevant to a disease model' for Rett syndrome, would be supported by the expression and agonist studies they did in the mouse and specific evidence terms could be used for those. The relationship itself seems to be capturing the model aspect of things.

If Gene G is a human gene - then I find the relationship stated above more confusing. 
It would make more sense to me as something like Gene G is 'implicated in a disease model' - or something to that effect.

Thanks,
Michelle
